python/python/tests/test_transaction.py

```diff
@@ -499,3 +499,30 @@ async def test_transaction_chaining_calls(self, redis_client: TRedisClient):
         transaction.set(key, "value").get(key).delete([key])
 
         assert await redis_client.exec(transaction) == [OK, "value", 1]
+
+    # object_freq is not tested in transaction_test as it requires that we set and later restore the max memory policy
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_transaction_object_freq(
```

Comment:
Can we union this test with other OBJECT * transaction tests?

Reply:
The other OBJECT commands do not require us to set and later restore the config with config_set, so they work fine in transaction_test with the rest of the commands. Do you still want me to group them together with this one?

Reply:
Please disregard this in that case

Reply:
Discussed with Yury and we won't group this with other OBJECT commands since this is the only one that requires the config to be set